demo/campaign/src/components/IndentedSectionGroup.tsx

```diff
@@ -7,18 +7,31 @@ export interface IndentedSectionGroupProps extends React.ComponentProps<typeof M
     cssClass?: string;
 }
 
-const CONTENT_WIDTH = theme.mailSize.contentWidth - theme.mailSize.contentSpacing * 2;
+export const indentedSectionGroupStyles = (
+    <MjmlStyle>{`
+      @media (max-width: 759px) {
```

Comment:
The default mail-width is `600px`. 
At this size and above, the default inline-styling should be all that's needed, as mail-clients do not reliably apply css. 

So the responsive behavior should only be applied below the default mail width, in this case `599px`. 

```suggestion
      @media (max-width: ${theme.mailSize.mailWidth - 1}px) {
```